packages/babel-cli/src/babel-external-helpers.js

```diff
@@ -1,9 +1,15 @@
+// @flow
+
 import commander from "commander";
+import isString from "lodash/isString";
 import { buildExternalHelpers } from "@babel/core";
 
-function collect(value, previousValue): Array<string> {
+function collect(
+  value: String | any,
```

Comment:
This isn't a string object, it's a `string`

Reply:
Oh yes. Sorry, that is my mistake. Will fix